src/DataWrangling/metadata_field.jl

```diff
@@ -95,7 +95,7 @@ function Field(metadata::Metadatum, arch=CPU();
         data = ds[dsname][:, :, :, 1]
 
         # Many ocean datasets use a "depth convention" for their vertical axis
-        if reversed_vertical_axis(metadata)
+        if reversed_vertical_axis(metadata.dataset)
```

Comment:
@navidcy this was the issue

Reply:
something else might be going on because the values don't seem to change for Tt[1] and Tt[2], right?